app/views/layouts/_navbar.html.erb

```diff
@@ -29,7 +29,22 @@
 
       <ul class="nav navbar-nav navbar-right">
           <p class="navbar-text">Signed in as <strong><%= current_user.username %></strong></p>
-
+          <% if current_user.admin? %>
+            <li class="dropdown">
+              <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Admin Tools<span class="caret"></span></a>
+              <ul class="dropdown-menu">
+                <li><%= link_to "Approve Recent Company Submissions", companies_path %></li>
+                <li><%= link_to "Update Existing Company Information", cities_path %></li>
```

Comment:
Yeah... All these design decisions are just placeholders... None of the routes are built.

More details are on the card in the pivotal tracker